### PR TITLE
Create provider component

### PR DESCRIPTION
**Pull request by djabarovgeorge**

### CODE REVIEW

File: apps/web/src/pages/integrations/components/ChannelGroup.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Grid } from '@mantine/core';
+import { ProviderCard } from './ProviderCard';
+import { Title } from '../../../design-system';
+
+export function ChannelGroup({ title, providers }: { providers: any[]; title: string }) {
```
Comment (scopsy) [inline apps/web/src/pages/integrations/components/ChannelGroup.tsx:6]: Do we have an interface for the providers here?

Comment (scopsy) [inline apps/web/src/pages/integrations/components/ChannelGroup.tsx:6]:
```suggestion
export function ChannelGroup({ title, providers }: { providers: ISharedProvider[]; title: string }) {
```

**Review by scopsy — APPROVED** [review]
A small note on the typescript interface

File: apps/web/src/pages/integrations/components/ChannelGroup.tsx (hunk 1 above)
Comment (djabarovgeorge) [reply apps/web/src/pages/integrations/components/ChannelGroup.tsx:6]: We do not, however, i added one on #380

Comment (djabarovgeorge) [reply apps/web/src/pages/integrations/components/ChannelGroup.tsx:6]: Same goes here, it resolved on #380

**Pull request merged**